react/features/base/media/middleware.js

```diff
@@ -68,6 +68,32 @@ function _setRoom({ dispatch, getState }, next, action) {
     const state = getState();
     const { room } = action;
     const roomIsValid = isRoomValid(room);
+    const { error } = state['features/base/conference'];
```

Comment:
the error can also come from the connection

Reply:
How do you mean?

Reply:
But does it cause any issues in this logic? Is it still okay to reuse these values?